stays/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from customers.models import Person, Provider, Pet
+
+
+class Stay(models.Model):
+    owner = models.ForeignKey(Person, on_delete=models.CASCADE)
+    provider = models.ForeignKey(Provider, on_delete=models.CASCADE)
+    start_date = models.DateField()
+    end_date = models.DateField()
```

Comment:
Shouldn't those fields be required ? Also i would add a default value for start_date

Reply:
DateField() is not required by default?

Reply:
It is, my bad!